lib/engines/content_block_manager/app/views/content_block_manager/content_block/shared/_schedule_publishing.html.erb

```diff
@@ -96,13 +96,23 @@
     <% end %>
     </div>
     <div>
-      <%= render partial: "content_block_manager/content_block/shared/cancel_delete_button",
-                 locals: {
-                   url: content_block_manager.content_block_manager_content_block_edition_path(
-                     @content_block_edition,
-                     redirect_path: content_block_manager.content_block_manager_content_block_document_path(@content_block_edition.document),
-                     ),
-                 } %>
+      <% if is_rescheduling %>
```

Comment:
open to a different naming or method for this! or turning into a component so it's easier to test?

Reply:
Hmmm, maybe a component would be good as we're using it in multiple places. Would make it more testable too.

Reply:
cool cool, I will be a bit less lazy and turn it into a component :)

Reply:
split out into quite a large a component, but I still think we need the feature tests in place to test the integration between the component and everything else - I've also noticed another bug so maybe there is more refactoring that could be done in this ticket  https://trello.com/c/aDIYdgfJ/837-bug-scheduled-publishing-form-does-not-populate-data-from-existing-block